packages/oas-utils/src/transforms/import-spec.test.ts

```diff
@@ -121,9 +185,111 @@ describe('Import OAS Specs', () => {
       const authScheme = res.securitySchemes[5] as SecuritySchemeOauth2
       expect(authScheme['x-scalar-client-id']).toEqual(testId)
     })
+
+    it('handles empty security requirements', async () => {
+      const specWithEmptySecurity = {
+        ...galaxy,
+        security: [{}],
```

Comment:
this is not an empty requirement btw, it is an optional one 😉 `[]` would be empty

Reply:
not that it matters